recipes/pdal/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class PdalConan(ConanFile):
+    name = "pdal"
+    description = "PDAL is Point Data Abstraction Library. GDAL for point cloud data."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://pdal.io"
+    license = "BSD-3-Clause"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True],  # disallow static builds
```

Comment:
Why? False value is handled in `configure()`

Keep `[False, True]` or remove this `shared` option. I would keep it, and eventually improve the recipe to properly build static libs (I'm surprised it doesn't work if static).

Reply:
I had tried the Croydon's suggestion from the previous conversation. Seems it doesn't work. I prefer to keep both values too because pdal is whitelisted to be shared by default.